client/options.go

```diff
@@ -178,6 +182,7 @@ func newOptions() *options {
 		clockOpts:                             clock.NewOptions(),
 		instrumentOpts:                        instrument.NewOptions(),
 		consistencyLevel:                      defaultConsistencyLevel,
```

Comment:
Should we call this `writeConsistencyLevel` assuming it's used for write consistency purposes?